sql_generators/review_checker/templates/review_checker_microsurvey/query.sql

```diff
@@ -0,0 +1,103 @@
+WITH new_and_existing_users AS (
+  SELECT
+    submission_date,
+    client_id,
+    IF(DATE_DIFF(submission_date, first_seen_date, DAY) <= 27, 1, 0) AS new_user,
+    IF(DATE_DIFF(submission_date, first_seen_date, DAY) > 27, 1, 0) AS existing_user
+  FROM
+    `moz-fx-data-shared-prod.telemetry.clients_last_seen`
+  WHERE
```

Comment:
FYI that this doesn't contain all new clients - see `clients_first_seen_v2`. Might be fine depending on your use-case

Reply:
The table is `clients_last_seen` and I do not see a `v2` of this. My rationale behind using a view is that even if underlying table change the table will remain the same.